### ISSUE TITLE
Timestamp of last valid reading

### ISSUE THREAD

**Issue by Zugschlus** [jomjol/AI-on-the-edge-device#3224]
### The Feature

Hi,

please consider giving the timestamp (or the round number) of the last valid reading that was not recjected because of "rate too high" or "negative rate" in the status web page. That would give an easy hint about whether the model is stuck.

It would also be nice to have a rejection rate like "76 of the last 100 readings were rejected"

Greetings
Marc

**Comment by SybexX** [comment]
Have you tried the current rolling yet? https://github.com/jomjol/AI-on-the-edge-device/actions/runs/10690676061

**Comment by Zugschlus** [comment]
Yes, it doesn't show any of the additional data that would be nice.

![image](https://github.com/user-attachments/assets/36bcddf7-88b3-44d4-b977-620a3e61c0bc)
